lib/factory_inspector.rb

```diff
@@ -20,9 +20,11 @@ def self.factory_inspect(factory)
         hash[:type] = "dynamic"
       when FactoryGirl::Attribute::Association
         hash[:type] = "association"
+        name_and_traits = attr.factory
+        hash[:factory_name] = name_and_traits.shift.to_s
+        hash[:traits] = name_and_traits.map{ |tr| tr.to_s }
```

Comment:
Space missing to the left of {.<br>Pass `&:to_s` as an argument to `map` instead of a block.